lib/create_poll.dart

```diff
@@ -80,10 +90,18 @@ class _CreatePollState extends State<CreatePoll> {
 
   Widget textFormField() {
     return TextFormField(
-      controller: _textEditingController,
+      //controller: _textEditingController,
```

Comment:
Why commented out code?

Reply:
cleaned

Reply:
using onchanged callback, instead of controller